src/Form/DataTransformerResolver.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\AdminBundle\Form;
+
+use Sonata\AdminBundle\Admin\FieldDescriptionInterface;
+use Sonata\AdminBundle\Form\DataTransformer\BooleanToStringTransformer;
+use Sonata\AdminBundle\Form\DataTransformer\ModelToIdTransformer;
+use Sonata\AdminBundle\Model\ModelManagerInterface;
+use Symfony\Component\Form\DataTransformerInterface;
+use Symfony\Component\Form\Extension\Core\DataTransformer\DateTimeToStringTransformer;
+
+/**
+ * @author Peter Gribanov <info@peter-gribanov.ru>
+ */
+final class DataTransformerResolver
```

Comment:
Can you please add some extension points or how could I add a custom transformer to the resolver?

Reply:
Do you want to add the possibility of customization on a global level, and not on a specific one?

Reply:
Yes, you could add global list fields via an admin extension, so you should also have the possibility to add a global mapper

Reply:
I added the opportunity to customize global transformers by constructor and `addCustomGlobalTransformer` method in `DataTransformerResolver`.